department-of-veterans-affairs/vets-website
# Added title and h1 to page not found shared component

## Summary
Added title and h1 to page not found shared component. Also made the h1 the focus of the page, so screen readers can read the page properly. Also added margins around the top text to match the bottom sections.

## Related issue(s)
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/84588
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/85960

## Testing done
- Updated unit test
- Manual testing for screen reader
  - Turn on your screen reader and navigate to an invalid URL for an application using this page not found component (e.g. http://localhost:3001/my-health/sdfsdf). Verify that the heading is read by the screen reader.

## Screenshots
|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |   ![image](https://github.com/department-of-veterans-affairs/vets-website/assets/63804190/19afbba6-ec48-45e9-9a22-446149371942)     | ![localhost_3001_my-health_kj](https://github.com/department-of-veterans-affairs/vets-website/assets/63804190/49d3c844-22a7-4652-82a8-d0517e9743cb) |
| Desktop |    ![image](https://github.com/department-of-veterans-affairs/vets-website/assets/63804190/64dce015-e19c-4b30-9368-f4341c14ea63)    |    ![image](https://github.com/department-of-veterans-affairs/vets-website/assets/63804190/bc03f5e4-9737-403f-8364-e892cc37980a)   |

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

## Acceptance criteria

### Quality Assurance & Testing


### Error Handling


### Authentication


### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:


## Requested Feedback
